packages/components/src/button/style.scss

```diff
@@ -242,6 +242,9 @@
 	&.is-secondary.is-busy:disabled,
 	&.is-secondary.is-busy[aria-disabled="true"] {
 		animation: components-button__busy-animation 2500ms infinite linear;
+		@media (prefers-reduced-motion: reduce) {
```

Comment:
I think you could use `@include reduce-motion("animation");` instead of this whole query.

Reply:
@jasmussen thanks, as I mentioned in this PR description:
> Note that is not possible to use the reduce-motion mixin because it sets `animation-duration: 1ms` but it doesn't reset `animation-iteration-count` which is `infinite` for this specific animation. I don't know why the reduce-motion mixin doesn't reset all the animation properties to 0 but it would be better to revisit it to have a mixin that is guaranteed to work with all animations / transitions.

Reply:
Oh I missed this. Can we update the mixin? I don't see why we shouldn't. If you do, be sure to restart npm run dev for the mixin to take.

Reply:
> Oh I missed this. Can we update the mixin? 

Yes of course, but I'd recommend a separate PR. I did already open https://github.com/WordPress/gutenberg/issues/55566 in fact.

Reply:
Sounds good to me. At most we could add an inline comment suggesting that we should refactor it to using `@include reduce-motion("animation");` as soon as https://github.com/WordPress/gutenberg/issues/55566 is closed